### PR TITLE
Add --idle-timeout argument to cypher shell

### PR DESCRIPTION
**Pull request by loveleif**
Docs of https://github.com/neo-technology/neo4j/pull/26148.

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1714.surge.sh

Updated pages:
- [Cypher Shell](https://neo4j-docs-operations-1714.surge.sh/operations-manual/5/tools/cypher-shell)

File: modules/ROOT/pages/tools/cypher-shell.adoc [hunk 1]
```diff
@@ -114,6 +115,10 @@ It can also be set using the environmental variable NEO4J_CYPHER_SHELL_HISTORY.
 |--notifications
 |Enable query notifications in interactive mode.
 |false
+
+|--idle-timeout IDLE-TIMEOUT
+|Idle timeout, when set the application will exit after the specified amount of idle time in interactive mode. Specify with `<hours>h<minutes>m<seconds>s`, for example `1h` (1 hour), `1h30m` (1 hour 30 minutes), `30m` (30 minutes).
```
Comment (renetapopova) [inline modules/ROOT/pages/tools/cypher-shell.adoc:120]:
```suggestion
|Closes the application after the specified amount of idle time in interactive mode. You can specify the duration using the format `<hours>h<minutes>m<seconds>s`, for example `1h` (1 hour), `1h30m` (1 hour 30 minutes), or `30m` (30 minutes).
```

Comment (renetapopova) [reply modules/ROOT/pages/tools/cypher-shell.adoc:120]: If you use this description, please also update the code.

**Review by renetapopova — COMMENTED** [review]
Thanks for documenting this option. I added a suggestion.

File: modules/ROOT/pages/tools/cypher-shell.adoc (hunk 1 above)
Comment (loveleif) [reply modules/ROOT/pages/tools/cypher-shell.adoc:120]: Sweet, thanks!

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1714.surge.sh

Updated pages:
- [Cypher Shell](https://neo4j-docs-operations-1714.surge.sh/operations-manual/5/tools/cypher-shell)

**Review by renetapopova — APPROVED** [review]
Thanks, @loveleif!

**Pull request merged**